en/docs/Learn/APIMonetization/monetizing-an-api.md

```diff
@@ -0,0 +1,703 @@
+
+1.  <a name> additionalProp </a> Configure the additional properties that are specific to the billing engine with regard to monetization in WSO2 API Manager.  
+     
+     After saving these configurations, the properties get populated in the Monetization enable/disable page on the Publisher in WSO2 API Manager.  
+
+    1.  Add the following configuration under Monetization in the `<API-M_HOME>/repository/resources/conf/templates/repository/conf/api-manager.xml.j2` file.
+
+        ``` java
+        <AdditionalProperties>
+        {% for origin in apim.monetization.additional_properties %}
+                <Property>{{origin}}</Property>
+        {% endfor %}
+        </AdditionalProperties>
+
+        {% if apim.monetization.additional_attributes is defined %}
+                    <AdditionalAttributes>
+                        {% for attribute in apim.monetization.additional_attributes %}
+                        <Attribute required="{{attribute.required}}">
+                            <Name>{{attribute.name}}</Name>
+                            <DisplayName>{{attribute.display_name}}</DisplayName>
+                            <Description>{{attribute.description}}</Description>
+                            <Default>{{attribute.default}}</Default>
+                        </Attribute>
+                        {% endfor %}
+                    </AdditionalAttributes>
+        {% endif %}
+        ```
+
+    2.  Add the following configuration in the `<API-M_HOME>/repository/conf/deployment.toml` file.
+
+        ``` java
+        [apim.monetization]
+        additional_attributes=['ConnectedAccountKey']
+        ```
+
+        When using Stripe as the billing engine, you need to enter the connect ID, which is the ID that indicates the link between the tenant admin and API provider Stripe accounts.
+
+2.  Enable API monetization.
+
+    1.  Sign in to the WSO2 API Manager Publisher.  
+        `https://<hostname>:9443/publisher-new`
+    2.  Click on the API that you wish to monetize.  
+        ![Enable monetization](../../assets/img/Learn/enable-monetization.png)
+    3.  Click **Monetization** to go to the Monetization configurations.
+    4.  Enter the connect ID as the connected account key and click
+        **Save**.  
+         
+         This creates the products and plans in Stripe based on the commercial tiers attached to the API.
+
+#### Step 3 - Subscribe to a monetized API
+
+[Subscribe to an API](../../Learn/Tutorials/subscribe-to-an-api) and invoke the API. When subscribing to an API, the price of the tier and billing plan is shown. Therefore, the Subscriber can select an appropriate plan and subscribe to it. 
+
+At the time of subscribing to an API, a Stripe customer is created in the Stripe platform account (e.g., the Stripe account for tenant admin). The following screenshot shows the customer record in the platform Stripe account.
+
+![Customer created in stripe](../../assets/img/Learn/customer-created-in-stripe.png)
```

Comment:
Image path needs to be corrected.

Reply:
Fixed.
I added the path based on the example on page 6 in the "Guidelines for Github Documentation" document. Looks like I misinterpreted the example.